packages/server/src/events/event.entity.ts

```diff
@@ -2,25 +2,25 @@ import { Table, Model, Column, DataType } from 'sequelize-typescript'
 
 @Table({ tableName: 'events' })
 export class Event extends Model {
-  @Column({ type: DataType.TEXT })
+  @Column({ type: DataType.CHAR(10) })
   block_number: string
-  @Column({ type: DataType.TEXT })
+  @Column({ type: DataType.CHAR(66) })
```

Comment:
contract hash = address = 42 chars

Reply:
done

Reply:
I also changed all the columns with addresses to be 42 char